pages/Home.jsx

```diff
@@ -0,0 +1,38 @@
+import {useEffect, useState} from "react";
+import {getArticles, getTopics} from "../services/API.js";
+import {ArticleRow} from "../components/ArticleRow.jsx";
+
+export const Home = () => {
+    const [articles, setArticles] = useState({});
+    const [loading, setLoading] = useState(true);
+    const [topics, setTopics] = useState([]);
+
+    useEffect(() => {
+        const getTopicList = async () => {
+            const topicData = await getTopics();
+            setTopics(topicData.topics);
+
+            const articleData = await getArticles();
+            const sortedArticles = {};
+            sortedArticles.latest = articleData.articles.sort((a, b) => new Date(b.created_at) - new Date(a.created_at)).slice(0, 4);
+            for (const topic of topicData.topics) {
```

Comment:
Please make sure to use your API to get articles by a certain topic instead of filtering here at the frontend.